t/app/controller/waste_kands_bulky.t

```diff
@@ -695,6 +657,138 @@ FixMyStreet::override_config {
+
+    $echo->mock(
+        'FindPoints',
+        sub {
+            [   {   Description => '2/3 Example Street, Sutton, SM2 5HF',
+                    Id          => '12345',
+                    SharedRef   => { Value => { anyType => 1000000002 } }
+                },
+            ]
+        }
+    );
+    $echo->mock('ReserveAvailableSlotsForEvent', sub {
+        my ($self, $service, $event_type, $property, $guid, $start, $end) = @_;
+        is $service, 413;
+        is $event_type, 1636;
+        is $property, 12345;
+        return [
+        {
+            StartDate => { DateTime => '2023-07-01T00:00:00Z' },
+            EndDate => { DateTime => '2023-07-02T00:00:00Z' },
+            Expiry => { DateTime => '2023-06-25T10:10:00Z' },
+            Reference => 'reserve1==',
+        }, {
+            StartDate => { DateTime => '2023-07-08T00:00:00Z' },
+            EndDate => { DateTime => '2023-07-09T00:00:00Z' },
+            Expiry => { DateTime => '2023-06-25T10:10:00Z' },
+            Reference => 'reserve2==',
+        }, {
+            StartDate => { DateTime => '2023-07-15T00:00:00Z' },
+            EndDate => { DateTime => '2023-07-16T00:00:00Z' },
+            Expiry => { DateTime => '2023-06-25T10:10:00Z' },
+            Reference => 'reserve3==',
+        },
+    ] });
+
+    $echo->mock('GetPointAddress', sub {
+        my ($self, $id) = @_;
+        return {
+            Id => $id,
+            SharedRef => { Value => { anyType => '1000000002' } },
+            PointType => 'PointAddress',
+            PointAddressType => { Name => 'House', Id => 1 },
+            Coordinates => { GeoPoint => { Latitude => 51.354679, Longitude => -0.183895 } },
+            Description => '2/3 Example Street, Sutton, SM2 5HF',
+        };
+    });
+
+    subtest 'Sutton staff paye payment sends user confirmation email' => sub {
+        FixMyStreet::Script::Alerts::send_updates();
+        $mech->clear_emails_ok;
+        $mech->log_in_ok($sutton_staff->email);
+        $mech->get_ok('/waste/12345/bulky');
+        $mech->submit_form_ok;
+        $mech->submit_form_ok({ with_fields => { name => 'Bob Marge', email => $user->email }});
+        $mech->submit_form_ok(
+            { with_fields => { chosen_date => '2023-07-08T00:00:00;reserve4==;2023-06-25T10:20:00' } }
+        );
+        $mech->submit_form_ok(
+            {   with_fields => {
+                    'item_1' => 'BBQ',
+                    'item_photo_1' => [ $sample_file, undef, Content_Type => 'image/jpeg' ],
+                    'item_2' => 'Bicycle',
+                    'item_3' => 'Bath',
+                },
+            },
+        );
+        $mech->submit_form_ok({ with_fields => { location => 'in the middle of the drive' } });
+        $mech->submit_form_ok({ with_fields => { tandc => 1 } });
+        $mech->submit_form_ok({ with_fields => { payenet_code => '54321' } });
+        FixMyStreet::Script::Alerts::send_updates();
```

Comment:
Are we running alerts here to check no other email sent? The one email being sent/checked for is being sent by the payenet_code submission the line above.

Reply:
I didn't twig that was being sent anyway. As Sutton doesn't currently have a comment user, should I remove that - or should I leave it in anyway?

Reply:
Probably clearer to remove it I guess.